gril/ril_constants.h

```diff
@@ -372,6 +372,17 @@
 #define RIL_REQUEST_VOICE_RADIO_TECH 108
 #define RIL_REQUEST_SET_INITIAL_ATTACH_APN 111
 
+#define RIL_REQUEST_IMS_REGISTRATION_STATE 112
+#define RIL_REQUEST_IMS_SEND_SMS 113
+#define RIL_REQUEST_GET_DATA_CALL_PROFILE 114
+#define RIL_REQUEST_SET_UICC_SUBSCRIPTION  115
```

Comment:
Note also that in lollipop the number for this request is 122. See:
http://androidxref.com/5.1.0_r1/xref/hardware/ril/include/telephony/ril.h